Pod/Classes/URBNAccordionDataSourceAdapter.m

```diff
@@ -117,6 +117,22 @@ - (BOOL)sectionIsOpen:(NSInteger)section {
     return [self.sectionsToKeepOpen containsIndex:section] || [self.expandedSections containsIndex:section];
 }
 
+- (void)appendSectionWithItems:(NSArray *)newItems {
+    NSAssert(YES, @"Call appendSectionObject:items:expanded: instead");
```

Comment:
Couldn't you just call through to `appendSectionObject:` and throw an assert in there for the section object? I think you should be asserting that you actually have a section in there anyways.